lib/ssl/src/ssl_gen_statem.erl

```diff
@@ -1994,10 +2000,18 @@ maybe_add_keylog({_, 'tlsv1.3'}, Info) ->
         ServerTrafficSecret0 = keylog_secret(ServerTrafficSecret0Bin, Prf),
         ClientHSecret = keylog_secret(ClientHSecretBin, Prf),
         ServerHSecret = keylog_secret(ServerHSecretBin, Prf),
-        Keylog = [io_lib:format("CLIENT_HANDSHAKE_TRAFFIC_SECRET ~64.16.0B ", [ClientRandom]) ++ ClientHSecret,
-                  io_lib:format("SERVER_HANDSHAKE_TRAFFIC_SECRET ~64.16.0B ", [ClientRandom]) ++ ServerHSecret,
-                  io_lib:format("CLIENT_TRAFFIC_SECRET_0 ~64.16.0B ", [ClientRandom]) ++ ClientTrafficSecret0,
-                  io_lib:format("SERVER_TRAFFIC_SECRET_0 ~64.16.0B ", [ClientRandom]) ++ ServerTrafficSecret0],
+        Keylog0 = [io_lib:format("CLIENT_HANDSHAKE_TRAFFIC_SECRET ~64.16.0B ", [ClientRandom]) ++ ClientHSecret,
+                   io_lib:format("SERVER_HANDSHAKE_TRAFFIC_SECRET ~64.16.0B ", [ClientRandom]) ++ ServerHSecret,
+                   io_lib:format("CLIENT_TRAFFIC_SECRET_0 ~64.16.0B ", [ClientRandom]) ++ ClientTrafficSecret0,
+                   io_lib:format("SERVER_TRAFFIC_SECRET_0 ~64.16.0B ", [ClientRandom]) ++ ServerTrafficSecret0],
+        Keylog = case lists:keyfind(client_early_data_secret, 1, Info) of
+                     {client_early_data_secret, {Alg, EarlySecret}} ->
+                         ClientEarlySecret = keylog_secret(EarlySecret, Alg),
```

Comment:
Can we use Prf here instead of storing Alg?

Reply:
I think so